openslides/motions/views.py

```diff
@@ -256,6 +259,34 @@ def update(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @list_route(methods=['post'])
+    def sort(self, request):
+        """
+        Sort motions. Also checks sort_parent field to prevent hierarchical loops.
```

Comment:
Please extend docstring: Describe what data should be send (something with nodes) so that the sorting works properly.

Reply:
I'm honest: I do not know this excatly, too. This is taken 1:1 from the agenda. I'll have a closer look at this code, if the client implements sorting of agenda or motions. Maybe we decide to pass different options, accumulated data, or something else. I also didn't test this yet. Is this ok for you?

Reply:
Hm. I am not happy, but it would be ok. Please add a comment here that we plan to refactor the sorting on client side and then also refactor this view.

Reply:
Done.